### PR TITLE
Refresh button: add to pull-card

### LINKED ISSUE #258: New UI: Add refresh button for each pull
In the old UI there's a button that tells the backend to "refresh" the pull from Github's API.

### Do This
Add a similar refresh button to the new UI

Old UI (refresh in upper right):
image

### PR DESCRIPTION
**Pull request by danielbeardsley**
The old UI had a button to refresh pulls from the API. Let's add the same thing to the new UI. Also, add a brief highlight when pulls update.

Closes #258

### CODE REVIEW

File: frontend/src/pull-card/refresh.tsx
Comment (ardelato) [inline frontend/src/pull-card/refresh.tsx:20]: Since we are merging #264 into this branch, does this mean that this pull is blocked until #264 gets merged since #264 will override some of these settings? 

https://github.com/iFixit/pulldasher/blob/d1adef54ac2d5781e37569e1219f7c14270d2194/frontend/src/pull-card/refresh.tsx#L20

File: frontend/src/pull-card/refresh.tsx
Comment (ardelato) [inline frontend/src/pull-card/refresh.tsx:30]: When it comes to setting the colors, do we usually want to opt for colors defined in the `*.less` files?

File: frontend/src/pull-card/refresh.tsx
Comment (ardelato) [inline frontend/src/pull-card/refresh.tsx:39]: I am not sure about naming this arrow function that same as the event type, what about renaming it to `handleOnClick`

**Comment by ardelato** [comment]
CR 👍 with questions

File: frontend/src/pull-card/refresh.tsx
Comment (danielbeardsley) [reply frontend/src/pull-card/refresh.tsx:20]: Other way around... That one should kinda be blocked on this one. Either way, it shouldn't matter that much, either can be merged first, it'll just make this diff larger though.

File: frontend/src/pull-card/refresh.tsx
Comment (danielbeardsley) [reply frontend/src/pull-card/refresh.tsx:30]: Thanks fro prompting, I looked into it this. We could make a new `--refresh-active-color: @white` but It's used *just* during the click operation (i.e. sub-1s) and it's gonna be white in both cases, I'd rather not bloat the theme with unimportant distinctions that don't need to change between dark/light.

**Comment by ardelato** [comment]
CR 👍

**Review by evannoronha — APPROVED** [review]
CR 🔥

**Comment by deltuh-vee** [comment]
QA 🎬 
* Pull card is highlighted when refreshing.  
* Button to refresh pull cards works.

**Pull request merged**
